packages/field-base/index.js

```diff
@@ -1,24 +1,18 @@
-export { ActiveMixin } from './src/active-mixin.js';
```

Comment:
Please update `package.json` for the `field-base` package to include the `component-base` dependency.